apps/galleries_institutions/components/filter_facet/locationSynonyms.coffee

```diff
@@ -0,0 +1,4 @@
+module.exports = [
+  ['st', 'saint']
```

Comment:
Got it! 👍  I was thinking if single target is used for matching, maybe we could map all the synonyms to the same value like
```javascript
{
  saint: 'st',
  san: 'st',
  santo: 'st',
  santa: 'st',
  harbour: 'harbor'
}
```
but looks like I am over simplifying stuff here. 😝